CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Note: Breaking changes between versions are indicated by "💥".
 
 ## Unreleased
 
+- [Security] Fix open redirect vulnerability in inactive user flow (see [commit](https://github.com/rgraber/edx-platform/commit/b726180aa26f928e89777240e3f8104f57967419))
```

Comment:
Can you release this as `v13.1.8`?

I think you just need to add an entry here and bump the version in `__about__.py`. @regisb 's automation should take care of the rest.

Reply:
Yes, it would make my life much easier if you could add a commit with the version bump. Please check the instructions here: https://docs.tutor.overhang.io/tutor.html#releasing-a-new-version
And this commit for inspiration: https://github.com/overhangio/tutor/commit/43c5177187bd7ddfa5a1ad995fe3537b75cd8851